requirements/doc.txt

```diff
@@ -1,17 +1,23 @@
 #
-# This file is autogenerated by pip-compile with Python 3.8
+# This file is autogenerated by pip-compile with Python 3.9
 # by the following command:
 #
-#    make upgrade
+#    pip-compile --output-file=requirements/doc.txt requirements/doc.in
```

Comment:
We should not compile a single file only as it causes inconsistencies in requirements. Instead, we can upgrade a single requirement by temporarily removing the `--upgrade` flag from `pip compile` commands within the make upgrade target.